README.md

```diff
@@ -3,3 +3,90 @@
 HyperUI is a collection of free Tailwind CSS components that can be used in your next project. With a range of components, you can build your next marketing website, admin dashboard, eCommerce store and much more.
 
 ![](https://hyperui.dev/og.jpg)
+
+## Table of Contents
+- [Getting Started](#getting-started)
+- [Usage](#usage)
+- [Components](#components)
+    - [🤖 Application UI](#application-ui)
+    - [📣 Marketing](#marketing)
+    - [🛍️ eCommerce](#ecommerce)
+- [Contributing](#contributing)
+- [License](#license)
+
+## Getting Started
+There is no HyperUI installation, if you have a Tailwind CSS project, you can copy the code and paste it into your project. Some components do use Alpine JS.
+
+## Usage
+Browse the website for the components you need, copy the code and paste it into your project. Some components will have support for dark mode and Alpine JS variants.
+
+1. Browse the website for a component you like.
+2. Preview the component at different breakpoints and check its compatibility with:
+    - LTR (left-to-right)
```

Comment:
I don't think we need the "(left-to-right)" part. I feel people in web development should know what LTR mean.